### PR TITLE
Fixes #5616 : Unable to translate content 

### LINKED ISSUE #5616: Unable to translate content
branch: 1.9.x
When trying to translate a content, I can't select translation localization.
Localization is forced to original's one.

Issue introduced by fa09e2e83a95d99eafb5afaf557eab3f74e86e9f.

### PR DESCRIPTION
**Pull request by TFleury**
Use a flag to allow selecting translation localization.

### CODE REVIEW

File: src/Orchard.Web/Modules/Orchard.Localization/Views/EditorTemplates/Parts/Localization.ContentTranslations.Edit.cshtml [hunk 1]
```diff
@@ -7,7 +7,7 @@
     <label for="@Html.FieldIdFor(m => m.SelectedCulture)">@T("Content Localization")</label>
     <div>
         @*Brand new content item*@
-        @if (Model.ContentItem.ContentItem.Id == 0) {
+        @if (Model.ContentItem.ContentItem.Id == 0 || (bool)(ViewData["Translating"] ?? false)) {
```
Comment (sebastienros) [inline src/Orchard.Web/Modules/Orchard.Localization/Views/EditorTemplates/Parts/Localization.ContentTranslations.Edit.cshtml]: I think you need to remove this

File: src/Orchard.Web/Modules/Orchard.Localization/Controllers/AdminController.cs [hunk 2]
```diff
@@ -50,13 +50,11 @@ public class AdminController : Controller, IUpdateModel {
...
 
-            var contentItemTranslation = _contentManager.New<LocalizationPart>(masterContentItem.ContentType);
-            contentItemTranslation.MasterContentItem = masterContentItem;
-
             // build the editor using the master content item so that
             // the form is pre-populated with the original values
-
+            
             var content = _contentManager.BuildEditor(masterContentItem);
+            ViewData["Translating"] = true;
```
Comment (sebastienros) [inline src/Orchard.Web/Modules/Orchard.Localization/Controllers/AdminController.cs]: I think you can remove this

**Comment by TFleury** [comment]
I agree on the fact that it's not very clean, but I didn't found a better way to know if we are editing or translating the original content item.

We could also look at RouteValues...

**Comment by TFleury** [comment]
Another way should be to *Clone* MasterContentItem and update LocalizationPart (reset culture and set MasterContentItem).

The main drawback with this method is that a new ContentItem is created in DB even if you don't save your changes.

**Comment by urbanit** [comment]
You are a life saver @TFleury ;-)

**Comment by sebastienros** [comment]
See issue for more details on how to actually fix the issue. I actually ended up with the same conclusion as you. Thanks.

**Pull request closed without merging**